core/mlang2.js

```diff
@@ -711,7 +711,7 @@ CLASS({
     reduceI: function(other) {
       for ( var i in other.groups ) {
         if ( this.groups[i] ) this.groups[i].reduceI(other.groups[i]);
-        else this.groups[i] = other.groups[i].deepClone();
```

Comment:
Shouldn't this still be deepClone()?  reduceI should merge the results from both the other expression into this one.